### ISSUE TITLE
The review entry form expands too much

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#6344]
### Describe the problem and steps to reproduce it:

- Sign in
- Rate an add-on
- Begin writing a review
- Enter a lot of text

### What happened?

The review entry form expands too much

<details>
<summary>Example</summary>

![screenshot_2018-09-21 octopar-troose-jagle-new-name add-ons for firefox](https://user-images.githubusercontent.com/55398/45892432-be90d000-bd8d-11e8-938c-af2d4b431d0a.png)

</details>

### What did you expect to happen?

After 600 characters, the review form should stop expanding. Scrollbars should appear.

### Anything else we should know?

Take a look at how [`AddonReviewManager`](https://github.com/mozilla/addons-frontend/blob/f71a19331ba96d409ea4d91b0f89a65a383198c9/src/amo/components/AddonReviewManager/index.js) uses [`DismissibleTextForm`](https://github.com/mozilla/addons-frontend/blob/f71a19331ba96d409ea4d91b0f89a65a383198c9/src/ui/components/DismissibleTextForm/index.js).

**Comment by saranshbarua** [comment]
@kumar303 I would like to work on this if it's available.

**Comment by ioanarusiczki** [comment]
@kumar303 I verified the review text box on the add-on's detail page (where scrollbars appear after ~600 chars) but also other pages.  -with  FF 62(Win10) on AMO dev
Scrollbars will appear for a long text for reviews and developer replies on the profile page and reviews page too ~ after 10 written text lines (1000 chars maybe)

![10 lines](https://user-images.githubusercontent.com/33448286/47286435-5c7af300-d5f7-11e8-8430-923b998ce91d.gif)

I don't see it as a problem, but need your confirmation to mark this issue as fixed.
Thanks!

**Comment by ioanarusiczki** [comment]
@kumar303 I verified the review text box on the add-on's detail page (where scrollbars appear after ~600 chars) but also other pages.  -with  FF 62(Win10) on AMO dev
Scrollbars will appear for a long text for reviews and developer replies on the profile page and reviews page too ~ after 10 written text lines (1000 chars maybe)

![10 lines](https://user-images.githubusercontent.com/33448286/47286435-5c7af300-d5f7-11e8-8430-923b998ce91d.gif)

I don't see it as a problem, but need your confirmation to mark this issue as fixed.
Thanks!

**Comment by ioanarusiczki** [comment]
@kumar303 I verified the review on the add-on's detail page. Scrollbars appear ~ after 600 chars. 
I checked this for reviews and dev replies on the user profile page and reviews page too 
Scrollbars appear after ~10 text lines around 1000 chars or so but the text box is wider on these pages  I don't see it as an issue but let me know if that's alright with you. 

Scrollbars on the other pages
![10 lines](https://user-images.githubusercontent.com/33448286/47286632-2d18b600-d5f8-11e8-8b28-ffa890273c14.gif)

**Comment by ioanarusiczki** [comment]
@kumar303 I verified the review on the add-on's detail page. Scrollbars appear ~ after 600 chars.

I checked this for reviews and dev replies on the user profile page and reviews page too.
Scrollbars appear after ~10 text lines around 1000 chars or so but the text box is wider on these pages I don't see it as an issue but let me know if that's alright with you.

Scrollbars on the other pages 

![10 lines](https://user-images.githubusercontent.com/33448286/47287023-8c2afa80-d5f9-11e8-9195-ef6d25a60f15.gif)

**Comment by ioanarusiczki** [comment]
@kumar303 I verified the review on the add-on's detail page. Scrollbars appear ~ after 600 chars.

I also checked this for reviews and dev replies on the user profile page and reviews page too.
Scrollbars appear after ~10 text lines around 1000 chars or so but the text box is wider on these pages I don't see it as an issue but let me know if that's alright with you.

User profile & Reviews page

![10 lines](https://user-images.githubusercontent.com/33448286/47290338-97852280-d607-11e8-8abb-a74ebea8cbe7.gif)
